lib/tlIO/STBWrite.cpp

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2021-2023 Darby Johnston
+// All rights reserved.
+
+#define STBIW_WINDOWS_UTF8
```

Comment:
Just checking about these defines, they are different for reading and writing? The write doesn't need "STBI_NO_JPEG" and read doesn't need "STBIW_WINDOWS_UTF8"?

Reply:
The NO_JPEG, etc. don't do anything in the stb_image_write.h file.  However you are right that I missed the UTF8 define for stb_image.h.  I fixed it (Good catch!).